src/app/home/directives/home-series.component.ts

```diff
@@ -19,7 +19,10 @@ import { StoryModel } from '../../shared';
       <a [routerLink]="['series', id]">
         <publish-image [imageDoc]="logoDoc"></publish-image>
       </a>
-      <p class="count">{{count}} Stories</p>
+      <p class="count">
+        <span *ngIf="count === 0">0 Stories</span>
+        <a *ngIf="count > 0" [routerLink]="['search', { tab: 'stories', seriesId: id }]">{{count}} Stories</a>
+      </p>
```

Comment:
nitpicky: do we want to make this smart enough to say "1 Story" instead of "1 Stories" ? could mimic the `storyNoun` logic from `series.component.ts`